src/book/entities/BaseEntity.ts

```diff
@@ -31,78 +30,8 @@ export default class BaseEntity extends BE {
   @v.IsString()
     guid: string = crypto.randomUUID().substring(0, 31);
 
-  async save(options?: SaveOptions): Promise<this> {
-    const entity = await super.save(options);
-
-    if (this.queryClient && ['Account', 'Commodity'].includes(this.constructor.name)) {
-      updateCache({ queryClient: this.queryClient, entity });
-    }
-
-    return entity;
-  }
-
-  async remove(options?: SaveOptions): Promise<this> {
-    const entity = await super.remove(options);
-
-    if (this.queryClient && ['Account', 'Commodity'].includes(this.constructor.name)) {
```

Comment:
While it is an optimization, it adds some complexity and potential eventual consistency problems. Account and Commodity are "cheap" entities to retrieve every time so removing for now